src/app/marketplace/view/view-ad.component.html

```diff
@@ -37,17 +39,6 @@
         <label-value *ngIf="status" [label]="i18n.general.status">
           {{ status }}
         </label-value>
-        <label-value *ngIf="ad.price" [label]="i18n.ad.price">
-          <ng-container *ngIf="ad.promotionalPrice">
-            <span class="promo-price">
-              {{ ad.price | currency:ad.currency }}
-            </span>
-            {{ ad.promotionalPrice | currency:ad.currency }}
-          </ng-container>
-          <ng-container *ngIf="ad.promotionalPrice == null">
-            {{ ad.price | currency:ad.currency }}
-          </ng-container>
-        </label-value>
         <ng-container *ngIf="data.canEdit">
```

Comment:
I'll commit in Cyclos 4.13 shortly a change that doesn't send the publication period and status if they are not visible.
Nowadays we're always sending them, even when they shouldn't be visible.
So, please, change the condition to `*ngIf="data.publicationPeriod"`.
The same applies to status.